pkgs/development/tools/conftest/default.nix

```diff
@@ -2,27 +2,32 @@
 
 buildGoModule rec {
   pname = "conftest";
-  version = "0.12.0";
+  version = "0.14.0";
+
+  // Something subtle in the go sum db is causing every download to
+  // get a new sum (and thus breaking the hash). This disables the
+  // fetching of the sum from the go sum database.
```

Comment:
This seems to be breaking the build; I assume these are meant to be comments :P

Reply:
@c00w fixed-up this issue and some formatting in your commit. See https://github.com/NixOS/nixpkgs/pull/68135/commits/1e2ee2a9554e4fd545366c8e2ae29393dd20027d